flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -291,7 +291,10 @@ def add_floodwall(self, floodwall: FloodWallModel, measure_path=Path):
 
         # Add floodwall attributes to geodataframe
         gdf_floodwall["name"] = floodwall.name
-        gdf_floodwall["z"] = floodwall.elevation.convert(UnitTypesLength("meters"))
+        if (gdf_floodwall.geometry.type == "MultiLineString").any():
```

Comment:
I don't think that a MultiLineString can have multiple z values. It is still a single feature, but with multiple lines, and only a single z value can be connected to it.

I think you would like to have multiple features with different z values attributed to them.

Reply:
Yes, that's true. I am turning it into linestrings because hydromt-sfincs cannot handle MultLineStrings. These LineStrings will simply have the same z-value as the initial MultLineSrting